cluster/gce/util.sh

```diff
@@ -778,6 +778,14 @@ function construct-linux-kubelet-flags {
     flags+="  --image-credential-provider-bin-dir=${AUTH_PROVIDER_GCP_LINUX_BIN_DIR}"
   fi
 
+  # Conditionally add the graceful shutdown flags
```

Comment:
if you see line 805

>   else # For nodes
>    flags+=" ${NODE_KUBELET_TEST_ARGS:-}"

you can already pass custom flags to the kubelet in the jobs

Reply:
Okay, I've updated https://github.com/kubernetes/test-infra/pull/32728 to pass the custom flags using`NODE_KUBELET_TEST_ARGS`.